agent/proxycfg/snapshot.go

```diff
@@ -644,6 +646,8 @@ type configSnapshotAPIGateway struct {
 
 	TLSConfig structs.GatewayTLSConfig
 
+	APIGatewayConfigEntry *structs.APIGatewayConfigEntry
```

Comment:
This is `GatewayConfig` below. Probably added in my branch after you had it here

Reply:
Ah, merges.